packages/mobile/src/app/saga.ts

```diff
@@ -113,8 +113,9 @@ function* watchAppState() {
       const newState = yield take(appStateChannel)
       Logger.debug(`${TAG}@monitorAppState`, `App changed state: ${newState}`)
       yield put(setAppState(newState))
+      // place event here
```

Comment:
Add TODO?

Reply:
Good catch. Actually don't need this anymore because looks like segment gets the app state changes.